rocks-db/src/lib.rs

```diff
@@ -789,4 +789,52 @@ impl Storage {
         }
         cf_options
     }
+
+    #[cfg(feature = "integration_tests")]
+    pub async fn clean_db(&self) {
+        let column_families_to_remove = [
```

Comment:
maybe we can somehow reuse ‎Storage::create_cf_descriptors() function here? In order not to forget to update code in two places instead of one if we add new columns. It's just a guess, it may not be possible to reuse this function such way